go.mod

```diff
@@ -1,71 +1,52 @@
 module github.com/spectrocloud-labs/spectro-cleanup
 
-go 1.22.11
+go 1.23.0
```

Comment:
not blocking on this, but why not bump to go1.24?

Reply:
because we don't have a public spectro image for go1.24 🤷🏼‍♂️